addons/website/static/src/scss/website.scss

```diff
@@ -1589,6 +1589,22 @@ input[value*="data-oe-translation-initial-sha"] {
     @extend .o_text_content_invisible;
 }
 
+// Offcanvas menu for in page filters (e.g. product list)
+.o_website_offcanvas {
+    .o_searchbar_form .dropdown-menu {
+        right: 0;
```

Comment:
can we add a `end-0` class instead?

Reply:
Looks like this rule is actually useless as there is no `.o_searchbar_form` inside the offcanvas